data_structures/linked_list/singly_link_list_deletion.c

```diff
@@ -4,6 +4,7 @@
   when passed with a key of the node.
 */
 #include <stdio.h>
+#include <stdlib.h>
```

Comment:
Why is this library required? 🤔

Reply:
I performed dynamic memory allocation using malloc, which is a part of standard library.I need to use stdlib in order to declare standard library.

Reply:
As there is use of "malloc" (which is a part of standard library) for dynamic memory allocation in line 18 of insert function, I included stdlib header file.